curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9282cd547785258cecf2.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-17
 
 # --description--
 
-Using arrow syntax, create an implicit return and use addition assignment to set the `innerHTML` of `tasksContainer` to empty backticks.
+Using arrow syntax complete the forEach callback function. Inside the callback function body use an addition assignment to set the innerHTML of tasksContainer to empty backticks.
```

Comment:
```suggestion
Using arrow syntax complete the `forEach` callback function. Inside the callback function body use an addition assignment to set the `innerHTML` of `tasksContainer` to empty backticks.
```

Reply:
Hi @lasjorg 
Make the required changes 

Thanks